screens/Cart/totalQty.tsx

```diff
@@ -0,0 +1,45 @@
+import { View, Text, StyleSheet } from "react-native";
+import React from "react";
+import { useSelector } from "react-redux";
+
+const totalQty = () => {
+  const cart = useSelector((state: { cart: any }) => state.cart.cart.cart);
+
+  const getTotalQuantity = () => {
+    let totalQuantity = 0;
+    cart.forEach((item: { quantity: number }) => {
+      totalQuantity += item.quantity;
+    });
+
+    if(totalQuantity > 99){
+        return 99 + "+"
```

Comment:
We may want that 99 to be a string to prevent any weirdness with type conversions.  You can just return `"99+"` instead.